feature/platform/ate_tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -112,14 +108,15 @@ func TestMain(m *testing.M) {
 
 func TestHardwarecards(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
-
 	cases := []struct {
 		desc          string
 		regexpPattern string
 		cardFields    properties
+		componentType oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT
+		parent        oc.E_PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT
 	}{{
 		desc:          "Chassis",
-		regexpPattern: "^Chassis",
+		componentType: chassisType,
```

Comment:
it looks like every componentType used is the same as the card pType. Drop the componentType field, too?

Reply:
Done

Reply:
I still see the componentType field in some tests

Reply:
Removed all